#nlp  message

string  text    #请求nlp处理的文本
string  voice   #发音人